src/pages/landing/ebooks/components/_image-text.js

```diff
@@ -0,0 +1,151 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { Header, Text, Li } from 'components/elements'
+import checkIcon from 'images/common/ebooks/check-icon.png'
+import BackgroundPattern from 'images/svg/landing/ebook-intro-bg.svg'
+import device from 'themes/device'
+import { localize, Localize } from 'components/localization'
+
+const BackgroundWrapper = styled.div`
+    width: 100%;
+    height: 100%;
+    margin: 0;
+    padding: 50px 60px 80px;
+    background-image: url(${BackgroundPattern});
+    background-size: cover;
+    @media ${device.tabletL} {
+        padding: 5% 35px 10%;
+    }
+    @media ${device.tabletS} {
+        padding: 10% 15px 18%;
+    }
+`
+
+const MediaWrapper = styled.div`
+    display: flex;
+    align-items: flex-start;
+    max-width: 1200px;
+    margin: 0 auto;
+    flex-wrap: wrap;
+
+    img.intro-forex {
+        align-self: center;
+        margin-right: 82px;
+        object-fit: contain;
+    }
+    h3.intro-text {
+        font-size: 48px;
+        font-weight: bold;
+        line-height: 1.25;
+        color: var(--color-black-3);
+        margin-bottom: 20px;
+    }
+
+    @media ${device.tabletL} {
+        margin: 0 auto;
+
+        img.intro-forex {
+            margin: 0 auto 20px;
+            padding: 25px;
+        }
+        h3.intro-text {
+            font-size: 30px;
+            text-align: center;
+        }
+    }
```

Comment:
Let's avoid styling with these class selectors. Also note that header sizing should be controlled via the `type` prop. Anything if the font size and line-height configuration doesn't match to those in `typography.js`, feel free to bring it up with the designers to keep it consistent.